curvefs/src/client/warmup/warmup_manager.h

```diff
@@ -214,6 +215,9 @@ class WarmupManager {
         return ret;
     }
 
+    virtual void CollectMetrics(InterfaceMetric *interface, int count,
```

Comment:
why make this a virtual function?

Reply:
> why make this a virtual function?

Currently it is the warmup of s3. Considering that the later bs may also support warmup, it is considered to be written like this.